Source/ORTS.Settings/InputSettings.cs

```diff
@@ -356,8 +356,9 @@ static void InitializeCommands(UserCommandInput[] Commands)
             Commands[(int)UserCommand.ControlBlowerIncrease] = new UserCommandKeyInput(0x31);
             Commands[(int)UserCommand.ControlSteamHeatDecrease] = new UserCommandKeyInput(0x20, KeyModifiers.Alt);
             Commands[(int)UserCommand.ControlSteamHeatIncrease] = new UserCommandKeyInput(0x16, KeyModifiers.Alt);
-            Commands[(int)UserCommand.ControlSteamBoosterDecrease] = new UserCommandKeyInput(0x11, KeyModifiers.Shift);
-            Commands[(int)UserCommand.ControlSteamBoosterIncrease] = new UserCommandKeyInput(0x11, KeyModifiers.Alt);
+            Commands[(int)UserCommand.ControlSteamBoosterAirValve] = new UserCommandKeyInput(0x11, KeyModifiers.Alt);
```

Comment:
HI Peter,

OR policy says "Reserve the Alt modifier for debug operations." Assignments are running out, but we have made progress on sub-menu pop-ups.

Until that day, please consider changing these Alt keys to something else, such as:
Ctrl+B and Ctrl+B+Shift

Thanks,

Chris

Reply:
I have changed the controls to Ctrl keys as suggested.